test/remote/gateways/remote_ebanx_test.rb

```diff
@@ -436,4 +445,36 @@ def test_successful_purchase_with_stored_credentials_cardholder_not_initial
     response = @gateway.purchase(@amount, @credit_card, options)
     assert_success response
   end
+
+  def test_successful_purchase_with_network_token
+    response = @gateway.purchase(@amount, @network_token, @options)
+    assert_success response
+    assert_equal 'Accepted', response.message
+  end
+
+  def test_failed_purchase_with_invalid_network_token
+    @network_token.number = '5102026827345142'
+    response = @gateway.purchase(@amount, @network_token, @options)
+    assert_failure response
+    assert_equal 'Invalid card or card type', response.message
+    assert_equal 'NOK', response.error_code
+  end
+
+  def test_failed_purchase_with_invalid_network_token_expire_date
+    @network_token.year = nil
+    response = @gateway.purchase(@amount, @network_token, @options)
+    assert_failure response
+    assert_equal 'Field network_token_expire_date is invalid', response.message
+    assert_equal 'BP-DR-136', response.error_code
```

Comment:
Error codes for network tokens are not present in the [docs](https://developer.ebanx.com/docs/resources/errorCodes/).